source/common/network/utility.cc

```diff
@@ -547,7 +548,63 @@ Api::IoCallUint64Result Utility::readFromSocket(IoHandle& handle,
                                                 UdpPacketProcessor& udp_packet_processor,
                                                 MonotonicTime receive_time,
                                                 uint32_t* packets_dropped) {
+
+  if (handle.supportsUdpGro()) {
+    Buffer::InstancePtr buffer = std::make_unique<Buffer::OwnedImpl>();
+    Buffer::RawSlice slice;
+
+    const uint64_t maxPacketSizeWithGro = 65535;
+    const uint64_t num_slices = buffer->reserve(maxPacketSizeWithGro, &slice, 1);
+    ASSERT(num_slices == 1u);
+
+    IoHandle::RecvMsgOutput output(1, packets_dropped);
+
+    Api::IoCallUint64Result result =
+        handle.recvmsg(&slice, num_slices, local_address.ip()->port(), output);
+
+    if (!result.ok()) {
+      return result;
+    }
+
+    ENVOY_LOG_MISC(trace, "recvmsg bytes {}", result.rc_);
+
+    // Get gso_size
+    if (output.msg_[0].gso_size_ == 0u) {
+      // Skip gso segmentation and proceed as a single payload.
+      ENVOY_LOG_MISC(warn, "Unspecified or zero gso_size received.");
```

Comment:
If we expect to reach this branch, we should not add a non-rate-limited warning log, because it will spam the log file. I think trace is the most verbose log level, maybe change to that?

Reply:
Changed the log level to trace here!

Reply:
As per discussions with Dan, removed this log. Instead added gso_size info with an existing log!
```
    ENVOY_LOG_MISC(trace, "recvmsg bytes {} with gso_size as {}", result.rc_, gso_size);
```